### PR TITLE
feature/add ESlint and install mongoDB

### PR DESCRIPTION
**Pull request by SCavus**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -11,6 +12,10 @@
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
     "jade": "~1.11.0",
+    "mongodb": "^3.6.0",
```
Comment (dahfool) [inline backend/package.json:15]: Don't think we need mongodb yet, as have not connect to the database.

Comment (AlexandruBudaca) [reply backend/package.json:15]: Did you run the lint? because I run it and I have an error about the parse.

Comment (SCavus) [reply backend/package.json]: I have removed MongoDB.

File: backend/routes/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-var express = require('express');
+var express = require("express");
```
Comment (dahfool) [inline backend/routes/index.js:1]: Have you decided to move to double quotes, the FE  is using single.

**Pull request merged**